<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vatek_factory</class>
 <widget class="QMainWindow" name="vatek_factory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vatek_factory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="qtvUIListStorage" name="lisections">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stpages"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenImage"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionBuildImage"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionBuildImage">
   <property name="text">
    <string>Build Image</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtvUIListStorage</class>
   <extends>QListWidget</extends>
   <header>ui/base/qtv_ui_storage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
